artifactory/emptymanager.go

```diff
@@ -259,3 +259,8 @@ func (esm *EmptyArtifactoryServicesManager) Client() *rthttpclient.ArtifactoryHt
 func (esm *EmptyArtifactoryServicesManager) GetAllRepositories() (*[]services.RepositoryDetails, error) {
 	panic("Failed: Method is not implemented")
 }
+
+// Compile time check of interface implementation.
+// ArtifactoryServicesManager requires that *EmptyArtifactoryServicesManager implements ArtifactoryServicesManager, otherwise,
+// this will no longer compile and we will be on notice that it needs to be updated.
```

Comment:
I suggest rephrasing this description a bit:

// Compile time check of interface implementation.
// Since EmptyArtifactoryServicesManager can be used by tests external to this project, we want this project's tests to fail, if EmptyArtifactoryServicesManager stops implementing the ArtifactoryServicesManager interface.